tests/unittests/test_fabric.py

```diff
@@ -81,6 +87,5 @@ def object_properties(parent_object):
                 ("id", "some_id"),
                 ("status", "UPTODATE"),
                 ("area_weight", 0.0),
-                # ("draping_ud_coefficient", 4.32), # TODO: enable this check, see backend issue #778698
```

Comment:
@greschd , do you know why this line and comment was there under the read_only properties?

Reply:
@roosre setting the draping UD coefficient to a value outside of the valid range `[0, 1]` produces an error in the ACP GUI. However, this is not an exception, but goes via the logging system.
This is not captured in PyACP currently, so the `draping_ud_coefficient` is silently not changed.

The intent of this line was to check that this raises, once the logged errors are captured in the ACP gRPC server.

Putting this check in the `read_only` section seems like a bit of a hack. I think we should either create a separate section `invalid_values`, or a one-off test for this. However, in general I think this could also be checked _only_ in the backend API tests.